Homework 4

Julie8899 opened the pull request.

Julie8899 committed:
Create HW3.py

Please review my HW3

Julie8899 committed:
Update HW3.py

Julie8899 committed:
Update for Transformation

I am not sure my transformation is good or not. Please give some advice. Thanks!

Julie8899 committed:
Updated for Storage Level

Julie8899 committed:
Set tempview for result and set storage level

Julie8899 committed:
Updated HW3

Julie8899 committed:
changed temview from results_v to results based on comments

Julie8899 committed:
Get the transformer

Julie8899 committed:
DRAFT HW

Please Review

huongphamBDA commented on HW4/HW4.py:
```diff
@@ -0,0 +1,191 @@
+
+# import tempfile
+# from PIL import Image
+
```
You need to use the `def main()` function and `if __name__ == "__main__":`

Julie8899 replied:
I updated it. Thanks!

huongphamBDA commented on HW4/HW4.py:
```diff
@@ -0,0 +1,191 @@
+data_df = pandas.read_csv("suspectheartrates.csv")
+
+
+def is_column_cat(df, column_name):
```
I thought we need to say out loud whether the response is continuous or boolean, and whether each predictor is cat/cont. Do you mean if df[column_name].nunique() > 2, then that column is continuous?

Julie8899 replied:
Yes.

huongphamBDA commented on HW4/HW4.py:
```diff
@@ -0,0 +1,191 @@
+            plot_mean_sq_diff(
+                data_df, pred_col_name, res_col_name, weighted=True, n_bins=30
+            )
+            pass
```
I don't see you doing the Random Forest part, calculating p t values, and generating a table with variables and rankings

Julie8899 replied:
I just commit a new version but I have not generated a table with variables and ranking.

huongphamBDA commented on HW4/HW4.py:
```diff
@@ -0,0 +1,191 @@
+def plot_scatter(df, pred_col_name, res_col_name):
+    pred_values = df[pred_col_name].tolist()
+    res_values = df[res_col_name].tolist()
+
```
How about Logistic Regression when the response is boolean type? I think boolean is when the column has only two values (True or False; 0 or 1). If it has two random values other than 0 and 1, it's categorical and not boolean.

Julie8899 committed:
HW 4 updated

Julie8899 committed:
Update HW4.py

Sorry, I accidentally create a new homework file called Homework 4 which is outside of HW4 folder. I already updated the HW4 file code inside HW4 folder which is same as Homework 4 file outside HW4 folder.
